lib/src/_shared/DateTextField.jsx

```diff
@@ -84,7 +88,23 @@ export default class DateTextField extends PureComponent {
   }
 
   handleChange = (e) => {
-    const { format, invalidDateMessage } = this.props;
+    const {
+      format,
+      invalidDateMessage,
+      clearable,
+      onChange,
+      onClear,
+    } = this.props;
+
+    if (clearable && e.target.value === '') {
+      onChange(null);
```

Comment:
This will fire `onChange` twice - direct invocation with null here, and one more time in `onClear` (see [its implementation](https://github.com/dmtrKovalenko/material-ui-pickers/blob/develop/lib/src/_shared/PickerBase.jsx#L54)).
I would suggest to fire `onClear` only

Reply:
You are correct, I thought its possible to pass in a onClear function from outside. I will change that.